### PR TITLE
zh-translation: .\docs\root\configuration\listeners\network_filters\s…

### PR DESCRIPTION
**Pull request by vgbhfive**
…ni_dynamic_forward_proxy_filter.rst

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: .\docs\root\configuration\listeners\network_filters\sni_dynamic_forward_proxy_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #295 
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter.rst [hunk 1]
```diff
@@ -1,29 +1,20 @@
 .. _config_network_filters_sni_dynamic_forward_proxy:
 
-SNI dynamic forward proxy
+SNI 动态转发代理
 =========================
 
 .. attention::
 
-  SNI dynamic forward proxy support should be considered alpha and not production ready.
+  SNI 动态转发代理目前仅支持 alpha，生产并未就绪。
```
Comment (helight) [inline docs/root/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter.rst]:
```suggestion
  SNI 动态转发代理目前还在 alpha 阶段，还不能用在生产环境。
```

File: docs/root/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter.rst [hunk 2]
```diff
@@ -1,29 +1,20 @@
...
 
-Through the combination of :ref:`TLS inspector <config_listener_filters_tls_inspector>` listener filter,
-this network filter and the
-:ref:`dynamic forward proxy cluster <envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>`,
-Envoy supports SNI based dynamic forward proxy. The implementation works just like the
-:ref:`HTTP dynamic forward proxy <arch_overview_http_dynamic_forward_proxy>`, but using the value in
-SNI as target host instead.
+通过 :ref:`TLS 检查 <config_listener_filters_tls_inspector>` 网络监听过滤器和 :ref:`动态转发代理集群 <envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>` 的组合，Envoy 支持基于 SNI 的动态转发代理。
```
Comment (helight) [inline docs/root/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter.rst]:
```suggestion
通过 :ref:`TLS 检查 <config_listener_filters_tls_inspector>` 网络监听过滤器，这个网络过滤器和 :ref:`动态转发代理集群 <envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>` 的组合，Envoy 支持基于 SNI 的动态转发代理。
```

File: docs/root/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter.rst [hunk 3]
```diff
@@ -1,29 +1,20 @@
...
+该实现的机制与 :ref:`HTTP 动态转发代理 <arch_overview_http_dynamic_forward_proxy>` 类似，但使用 SNI 中的值作为目标主机。
 
-The following is a complete configuration that configures both this filter
-as well as the :ref:`dynamic forward proxy cluster
-<envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>`. Both filter and cluster
-must be configured together and point to the same DNS cache parameters for Envoy to operate as an
-SNI dynamic forward proxy.
+以下是完整的配置，该配置包含了过滤器和 :ref:`动态转发代理集群 <envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>` 。过滤器和集群都必须一起配置并且指向相同的 DNS 缓存参数，使 Envoy 作为 SNI 动态转发代理。
```
Comment (helight) [inline docs/root/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter.rst]:
```suggestion
以下是完整的配置，该配置包含了这个过滤器和 :ref:`动态转发代理集群 <envoy_api_msg_config.cluster.dynamic_forward_proxy.v2alpha.ClusterConfig>` 。过滤器和集群都必须一起配置并且指向相同的 DNS 缓存参数，以便让 Envoy 作为 SNI 动态转发代理。
```

**Review by helight — COMMENTED** [review]
一点建议

**Review by majinghe — APPROVED** [review]

**Comment by helight** [comment]
LGTM

**Pull request merged**